sdk/eventhub/event-hubs/README.md

```diff
@@ -181,6 +182,59 @@ for await (const events of consumer.getEventIterator()){
 }
 ```
 
+### Consume events from all partitions of an Event Hub
+
+To consume events for all partitions of an Event Hub, you'll create an `EventProcessor` for a specific consumer group. When an Event Hub is created, it provides a default consumer group that can be used to get started.
+
+The `EventProcessor` will delegate processing of events to a `PartitionManager` implementation that you provide, allowing your logic to focus on the logic needed to provide value while the processor holds responsibility for managing the underlying consumer operations. In our example, we will focus on building the `EventProcessor` and use a very minimal partition processor that does no actual processing.
+
+```javascript
+class SimplePartitionProcessor {
+  async processEvents(events: ReceivedEventData[]) {
+    // your code here
```

Comment:
It would be nice to show how to checkpoint as well.

Reply:
I am torn between keeping the sample simple and showing off all the features...